portfolio.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
this is a copy of your contact page, try adding multiple images and playing with the css.